Components/buttons.css

```diff
@@ -0,0 +1,117 @@
+.button {
+  padding: var(--space-xs);
+  outline: none;
+  border: 0px solid transparent;
+  border-radius: 0.2rem;
+  font-family: "Merriweather Sans", sans-serif;
+  display: inline-block;
+  font-size: 1rem;
+  cursor: pointer;
+  transition: all 0.25s ease;
+}
+.button-primary {
+  color: var(--white);
+  background: var(--primary);
+}
+.button-secondary {
+  color: var(--white);
+  background: var(--secondary);
+}
+.button-success {
+  color: var(--white);
+  background: var(--success);
+}
+.button-warning {
+  color: var(--black);
+  background: var(--warning);
+}
+.button-info {
+  color: var(--black);
+  background: var(--info);
+}
+.button-error {
+  color: var(--white);
+  background: var(--error);
+}
+.button-monochrome {
+  color: var(--white);
+  background: var(--dark);
+}
+.button-primary:hover ,.button-secondary:hover,.button-success:hover,.button-warning:hover,.button-info:hover,.button-error:hover,.button-monochrome:hover{
```

Comment:
You can create a hover class and assign it to reletive buttons. Instead of defining all the classes here and giving value.

Reply:
This was intentional so that user dont have to give a seperate class for hover effect in the html. In case I am missing something let me know.